### PR TITLE
Fix a minor error , enhance linting.

### PR DESCRIPTION
**Pull request by ahmedaabouzied**
**The main reason for this pull request is to fix the minor error in file `watchlist.go` with the `livemode` property type.** It should have been a `bool` not a `string` which resulted in failure of json `Unmasrshalling`.This failed on tests. **Now all tests pass** :)

I understand that this package was wrote about 5 years ago and the linting standards might have changed. I applied the following changes to make the code complied with Golint : 

1. Omit type int from declaration of var `apiVersion` as it will be inferred from the right-hand side. 
2. Rename all vars that has `Url` in them to `URL`.
3. Rename all vars with `Id` to `ID`.
4. Add comments before each exported function or struct describing what it is.
5. Remove any else after an if statement that ends with return

I hope you find this useful. I'm going to use this package in a project we are working on.

Thanks

### CODE REVIEW

File: candidate.go [hunk 1]
```diff
@@ -105,26 +111,29 @@ func (self *CandidateClient) Update(id string, params *CandidateParams) (*Candid
...
 
-func (self *CandidateClient) List() ([]*Candidate, error) {
-	return self.list(25, 0)
+// List returns last added 25 clients
+func (candidateClient *CandidateClient) List() ([]*Candidate, error) {
+	return candidateClient.list(25, 0)
 }
 
-func (self *CandidateClient) ListN(count, offset int) ([]*Candidate, error) {
+// ListN returns the first given count of clients. If given zero, it returns the first 25 clients by default.
```
Comment (connorjacobsen) [inline candidate.go]: If I remember correctly, and I very well may not, this is meant to be an offset based pagination method?

**Review by connorjacobsen — APPROVED** [review]
I'm happy to add you as a maintainer for this project if you'd like. Sorry for the delay by the way, it slipped passed me originally.

**Comment by connorjacobsen** [comment]
@ahmedaabouzied Depending on your usecase, it may also be worth looking into the newest products from the company (which rebranded as cognito): https://cognitohq.com/docs/identity-verification-api

**Comment by ahmedaabouzied** [comment]
@connorjacobsen Hey this is great . I didn't know about Cognito. Will definitely look into their API. Thanks

**Comment by connorjacobsen** [comment]
Cognito is the same company, they rebranded.

**Pull request merged**